python/graphstorm/sagemaker/sagemaker_partition.py

```diff
@@ -386,6 +392,75 @@ def data_dispatch_step(partition_dir):
 
         data_dispatch_step(local_partition_path)
 
+        # Leader instance copies raw_id_mappings from input to dist_graph output on S3
+        # using aws cli, usually much faster than using boto3
+        # Will also upload the updated GConstruct/GSProcessing config, if one exists
+        raw_id_mappings_s3_path = os.path.join(graph_data_s3, "raw_id_mappings")
+        logging.info("Copying raw_id_mappings from %s to %s",
+            raw_id_mappings_s3_path, s3_dglgraph_output)
+        # Copy raw_id_mappings from input to dist_graph output on S3
+        subprocess.call([
+            "aws", "configure", "set", "default.s3.max_concurrent_requests", "150"])
+        subprocess.check_call(
+            [
+                "aws",
+                "s3",
+                "sync",
+                "--only-show-errors",
+                "--region",
+                os.environ["AWS_REGION"],
+                raw_id_mappings_s3_path,
+                f"{s3_dglgraph_output}/raw_id_mappings"])
+
+        # Try to download the GSProcessing launch arguments JSON
+        launch_arguments_s3_path = os.path.join(graph_data_s3_no_trailing, "launch_arguments.json")
+        download_data_from_s3(
+            launch_arguments_s3_path,
+            graph_data_path,
+            sagemaker_session
+        )
+        launch_arguments_local = os.path.join(graph_data_path, "launch_arguments.json")
```

Comment:
launch_arguments of GSProcessing?

Reply:
Yes, this is how GSprocessing itself will name the file